debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleDatabaseVersionTest.java

```diff
@@ -19,63 +17,14 @@
 public class OracleDatabaseVersionTest {
 
     @Test
-    @FixFor("DBZ-7257")
-    public void shouldParseOracle11g() throws Exception {
-        String banner = "Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production";
-        OracleDatabaseVersion version = OracleDatabaseVersion.parse(banner);
-        assertOracleVersion(version, 11, 2, 0, 4, 0, banner);
-    }
-
-    @Test
-    @FixFor("DBZ-7257")
-    public void shouldParseOracle12c() throws Exception {
-        String banner = "Oracle Database 12c Enterprise Edition Release 12.1.0.2.0 - 64bit Production";
-        OracleDatabaseVersion version = OracleDatabaseVersion.parse(banner);
-        assertOracleVersion(version, 12, 1, 0, 2, 0, banner);
-    }
-
-    @Test
-    public void shouldParseOracle19c() throws Exception {
-        String banner = "Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production\nVersion 19.3.0.0.0";
-        OracleDatabaseVersion version = OracleDatabaseVersion.parse(banner);
-        assertOracleVersion(version, 19, 3, 0, 0, 0, banner);
-    }
-
-    @Test
-    public void shouldParseOracle21c() throws Exception {
-        String banner = "Oracle Database 21c Express Edition Release 21.0.0.0.0 - Production\nVersion 21.3.0.0.0";
-        OracleDatabaseVersion version = OracleDatabaseVersion.parse(banner);
-        assertOracleVersion(version, 21, 3, 0, 0, 0, banner);
+    public void shouldParseOracleVersion() {
+        String version = "23.4.0.23.10";
+        OracleDatabaseVersion oracleDatabaseVersion = OracleDatabaseVersion.parse(version);
+        assertThat(oracleDatabaseVersion.getMajor()).isEqualTo(23);
+        assertThat(oracleDatabaseVersion.getMaintenance()).isEqualTo(4);
+        assertThat(oracleDatabaseVersion.getAppServer()).isEqualTo(0);
+        assertThat(oracleDatabaseVersion.getComponent()).isEqualTo(23);
+        assertThat(oracleDatabaseVersion.getPlatform()).isEqualTo(10);
+        assertThat(oracleDatabaseVersion.getVersion()).isEqualTo(version);
```

Comment:
This test covers pretty much all version cases so replaced the others. Please raise a concern if any of the older ones might cause issues.